From 6a476898b89a3a1aaa728f399b14522af4939ed0 Mon Sep 17 00:00:00 2001
From: Yongjun Zhang <yzhang@cloudera.com>
Date: Tue, 17 May 2016 15:45:15 -0700
Subject: [PATCH 1564/2748] HDFS-10381, DataStreamer DataNode exclusion log
 message should be warning. (John Zhuge via
 Yongjun Zhang)

(cherry picked from commit 16c07cc68a3e0a06f57b7f4c7207cc8e5dce211f)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DataStreamer.java
	hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/StripedDataStreamer.java

Change-Id: I365ee059381ae40e5ac4c36fc6d68df53715a6ca
---
 .../org/apache/hadoop/hdfs/DFSOutputStream.java    |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSOutputStream.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSOutputStream.java
index 26f34fc..2e7ab27 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSOutputStream.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSOutputStream.java
@@ -1512,11 +1512,11 @@ private LocatedBlock nextBlockOutputStream() throws IOException {
         success = createBlockOutputStream(nodes, storageTypes, 0L, false);
 
         if (!success) {
-          DFSClient.LOG.info("Abandoning " + block);
+          DFSClient.LOG.warn("Abandoning " + block);
           dfsClient.namenode.abandonBlock(block, fileId, src,
               dfsClient.clientName);
           block = null;
-          DFSClient.LOG.info("Excluding datanode " + nodes[errorIndex]);
+          DFSClient.LOG.warn("Excluding datanode " + nodes[errorIndex]);
           excludedNodes.put(nodes[errorIndex], nodes[errorIndex]);
         }
       } while (!success && --count >= 0);
-- 
1.7.9.5

